flink-libraries/flink-table/src/main/scala/org/apache/flink/table/functions/AggregateFunction.scala

```diff
@@ -17,6 +17,8 @@
  */
 package org.apache.flink.table.functions
 
```

Comment:
Suggest to remove the useless java doc, something like:
1. line 35 `- getAccumulatorType.`
2. line 102 to line 112 `def getResultType: TypeInformation[_]`